config/config/common-config.yaml

```
@@ -1102,3 +1109,23 @@ nodeLocalDns:
     requests:
       cpu: 25m
       memory: 40Mi
+
+externalDns:
+  enabled: false
+  txtOwnerId: set-me-if-externalDns.enabled
+  sources:
```

```suggestion
  enabled: false
  provider: route53 # or awsRoute53
  txtOwnerId: set-me-if-externalDns.enabled
  sources:
```
Then fail templating if it is anything else with an error message saying it is only currently supported with AWS Route53.